src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayLedDefaultHDLGeneratorFactory.java

```diff
@@ -95,19 +99,20 @@
             .pair("outs", LedArrayGenericHDLGeneratorFactory.LedArrayOutputs);
 
     if (HDL.isVHDL()) {
-      contents.addLines(
-          "genLeds : FOR n in (nrOfLeds-1) DOWNTO 0 GENERATE",
-          "   {{outs}}(n) <= NOT({{ins}}(n)) WHEN activeLow = 1 ELSE {{ins}}(n);",
-          "END GENERATE;");
+      contents.add("""
+          genLeds : FOR n in (nrOfLeds-1) DOWNTO 0 GENERATE
+             {{outs}}(n) <= NOT({{ins}}(n)) WHEN activeLow = 1 ELSE {{ins}}(n);
+          END GENERATE;
+          """);
     } else {
-      contents.addLines(
-          "genvar i;",
-          "generate",
-          "   for (i = 0; i < nrOfLeds; i = i + 1)",
-          "   begin:outputs",
-          "      assign {{outs}}[i] = (activeLow == 1) ? ~{{ins}}[i] : {{ins}}[i];",
-          "   end",
-          "endgenerate");
+      contents.add("""
+          genvar i;
+          generate
+             for (i = 0; i < nrOfLeds; i = i + 1) begin
```

Comment:
As in PR #955 the 
```
for (i = 0; i < nrOfLeds; i = i + 1) begin
```

should be replaced by:
```
for (i = 0; i < nrOfLeds; i = i + 1) 
  begin:outputs
```
 for all the verilog versions of the LED*HDLGenerator and RGB*HDLGenerator

Reply:
> for all the verilog versions of the LEDHDLGenerator and RGBHDLGenerator

I am unable to find other occurences?

BTW: ClockHDLGeneratorFactory.java line 170:

```java
167     if (ClockTick)
168     begin
169        s_derived_clock_reg[0] <= s_derived_clock_reg[0] ^ s_counter_is_zero;
170        for (n = 1; n < {{phase}}; n = n+1) begin
171           s_derived_clock_reg[n] <= s_derived_clock_reg[n-1];
172        end
173     end
```